src/android/build/localnotification.gradle

```diff
@@ -26,21 +26,7 @@ if (!project.ext.has('appShortcutBadgerVersion')) {
     ext.appShortcutBadgerVersion = '1.1.19'
 }
 
-String getVersion() {
-  def compileSdkVersion = Integer.parseInt(project.android.compileSdkVersion.split("-")[1])
-  return compileSdkVersion >= 28 ? "28.0.0" : "26.+" 
-}
-
 dependencies {
     implementation "me.leolin:ShortcutBadger:${appShortcutBadgerVersion}@aar"
+    implementation 'androidx.media:media:1.6.0'
```

Comment:
Is there no need of update also this
import android.support.v4
in the
OutSystems/cordova-plugin-local-notifications/src/android/notification/Options.java
and in 
OutSystems/cordova-plugin-local-notifications/src/android/notification/Builder.java

Reply:
I saw it, is in the other PR

Reply:
I also thought we had to update those when I created the task, but it turns out that there is no import starting with `androidx` that contains the `MediaSessionCompat` class which is needed both in `Options.java` and `Builder.java`. 

However, by removing the dependency on the Support Library from the Gradle file, MABS 10 builds work. If we keep them, MABS 10 builds fail. So I believe we only need to remove them from the Gradle file.

Reply:
Yes make sense, MediaSessionCompact belongs to androidx also, [https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:media/media/src/main/java/android/support/v4/media/session/MediaSessionCompat.java?q=file:android%2Fsupport%2Fv4%2Fmedia%2Fsession%2FMediaSessionCompat.java%20class:android.support.v4.media.session.MediaSessionCompat](url)

Reply:
Yes exactly.